assets/css/headers/header-default.css

```diff
@@ -120,10 +120,30 @@
 	padding: 0 10px 10px;
 }
 
-@media(min-width: 768px) and (max-width: 998px) {
-	.header .navbar-brand { 
+.header .navbar-brand img {
+	padding-bottom: 4px;
+}
+
+@media(min-width: 992px) and (max-width: 1200px){
+	.header .navbar-brand {
+		white-space: pre-line;
+		top: -47px;
+		margin-bottom: -36px;
+	}
+}
+
+@media(max-width: 440px) {
+	.header .navbar-brand {
+		white-space: pre-line;
+		top: -42px;
+		margin-bottom: -36px;
+	}
+}
+
+@media(min-width: 767px) and (max-width: 998px) {
+	.header .navbar-brand {
 		margin-top: 0;
```

Comment:
Sure! These styling changes were meant to accomplish two things:
a) make the text in the logo and the text in the remainder of the header line up when inline
b) change the text in the header to display beneath the logo as appropriate when the window resizes